docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md

```diff
@@ -0,0 +1,300 @@
+|  Edit Inventory       | Create from existing<br />Create new<br />Move<br />Register<br />Remove<br />Unregister     |
+|  Guest Operations     | Alias modification<br />Alias query<br />Modify guest operations<br />Invoke programs<br />Query guest operations |
+|  Interaction          | Console Interaction<br />Power on/off |
+|  Provisioning         | Allow disk access<br />Allow file access<br />Allow read-only disk access<br />Allow VM download<br />Allow VM upload<br />Clone template<br />Clone VM<br />Create template from VM<br />Customize guest<br />Deploy template<br />Mark as template<br />Modify customization specifications<br />Promote disks<br />Read customization specifications |
+|  Service Configuration| Allow notifications<br />Allow polling of global event notifications<br />Manage service configurations<br />Modify service configurations<br />Query service configurations<br />Read service configurations |
+|  Snapshot Management  | Create snapshot<br />Remove snapshot<br />Rename snapshot<br />Revert to snapshot |
+|  vSphere Replication  | Configure replication<br />Manage replication<br />Monitor replication |
+|  vSAN  | Cluster<br />ShallowRekey |
+
+
+</TabItem>
+
+
+
+<TabItem label="6.0.x" value="6.0.x">
+
+
+| **vSphere Object**| **Privileges**                              |
+|-------------------|---------------------------------------------|
+| **CNS**               | Searchable                                  |
+| **Datastore**         | Allocate space<br />Browse datastore<br />Low-level file operations<br />Remove file<br />Update VM files<br />Update VM metadata |
+| **Folder**            | Create Folder<br />Delete folder<br />Move folder<br />Rename folder|
+| **Host**              | Local operations: Reconfigure VM            |
+| **Network**           | Assign network                              |
+| **Profile-driven storage** | Profile-driven storage view            |
+| **Resource**          | Apply recommendation<br />Assign VM to resource pool<br />Migrate powered off VM<br />Migrate powered on VM<br />Query vMotion |
+| **Sessions**          | Validate session                            |
+| **spectro-templates** | Read only. This is the vSphere folder created during the install. For airgap installs, you must manually create this folder.    |
+| **Storage views**     | View                                        |
+| **Tasks**             | Create task<br />Update task                |
+| **vApp**              | Import<br />View OVF environment<br />Configure vAPP applications<br />Configure vApp instances |
+| **vSphere tagging**   | Assign or Unassign vSphere Tag<br />Create vSphere Tag<br />Delete vSphere Tag<br />Edit vSphere Tag |
+
+
+
+The following table lists spectro role privileges for VMs by category. All privileges are for the vSphere object, Virtual Machines.
```

Comment:
The last sentence doesn't make sense to me: "All privileges are for the vSphere object, Virtual Machines."